itwin-platform-access/imodels-access-frontend/src/FrontendIModelsAccess.ts

```diff
@@ -84,4 +98,30 @@ export class FrontendIModelsAccess implements FrontendHubAccess {
       return PlatformToClientAdapter.toAuthorization(token);
     };
   }
+
+  private async _getChangesetFromLatestNamedVersion(arg: IModelIdArg): Promise<ChangesetIndexAndId> {
```

Comment:
same comment about the naming - private methods in this repo are not prefixed with _